src/main/java/pl/kubiczak/orrrg/test/orrrgtest/Hello.java

```diff
@@ -3,6 +3,11 @@
 public class Hello {
 
   String getMessage(String name) {
+
+    if (true) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)